sentences/cs/cover_HassTurnOff.yaml

```diff
@@ -11,6 +11,22 @@ intents:
           domain: cover
         response: garage
 
+      - sentences:
+          - "<zavrit> {name} [<area>]"
+        requires_context:
+          domain: cover
+        response: cover
+
+      - sentences:
+          - "<zatahnout> {name} [<area>]"
```

Comment:
is there reason to not allow both (zavrit|zatahnout)?

Reply:
`<zavrit>` is allowed above. I've allowed `<zatahnout>` only for blinds etc.

Reply:
I was wondering why there are two separate sentences.

What device in domain cover, but not with class blind, curtain or shutter should use zavrit and not zatahnout?

Reply:
window, door, gate ? However we should probably add others like shade or awning. I've just copy the list already used: https://github.com/home-assistant/intents/blob/main/sentences/cs/cover_HassTurnOff.yaml#L17-L21

We can add more device_classes for both `<zatahnout>` in separate PR.

Reply:
Thanks for explanation. Personally I would probably not put them in domain cover, but I see we anyway already have garage_door as cover. So let's go with this now and we can change a bit our fixtures later.